api/python/quilt3/packages.py

```diff
@@ -748,13 +748,21 @@ def set_dir(self, lkey, path=None, meta=None):
             path(string): path to scan for files to add to package.
                 If None, lkey will be substituted in as the path.
             meta(dict): user level metadata dict to attach to lkey directory entry.
+            update_policy(str): can be either 'incoming' (default) or 'existing'.
+                If 'incoming', whenever logical keys match, always take the new entry from set_dir.
+                If 'existing', whenever logical keys match, retain existing entries
+                and ignore new entries from set_dir.
 
         Returns:
             self
 
         Raises:
             When `path` doesn't exist
+            ValueError: if the 'update_policy' is not in PACKAGE_UPDATE_POLICY set.
```

Comment:
```suggestion
            ValueError: if the `update_policy` is not in PACKAGE_UPDATE_POLICY set.
```
I think we shouldn't refer to `PACKAGE_UPDATE_POLICY` here.
@akarve could you help with wording?